### PR TITLE
longer password

### PR DESCRIPTION
**Pull request by stephenwaite**
e2e test fail

### CODE REVIEW

**Comment by stephenwaite** [comment]
pretty sure this is whacked so will wait for Brady to confirm it's the new devil, github actions :)

**Comment by bradymiller** [comment]
Can't get to github from work, but guessing you had a e2e test fail saying
something about an alert showing up. That is a test that fails about 1 in
50 tests, so shows up every once in awhile. Recommend going to listing of
commits, clicking the red X, and then clicking on one of the Tests and then
at top right black box click on the pull down and rerun the tests. That
should clear it up.

File: tests/Tests/E2e/CreateStaffTest.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function check_add_user(): void
         $openEmrPage = $this->e2eBaseUrl;
         $client = static::createPantherClient(['external_base_uri' => $openEmrPage]);
         $lp = new LoginPage($client, $this);
-        $mp = $lp->login('admin', 'pass');
+        $mp = $lp->login('admin', 'password123');
```
Comment (bradymiller) [inline tests/Tests/E2e/CreateStaffTest.php:36]: This is failing all e2e testing since it needs to be the actual password of the admin user.

**Comment by bradymiller** [comment]
I just reran the tests on the main commit, so master is now passing. The e2e functional testing has some issues (where there is a low rate of fails happening for unclear reasons). May make sense to make an issue to catalog them (there are now 2) since @spyrkob is working on one of them here:
https://github.com/openemr/openemr/pull/4071

The one @spyrkob is looking into happens about every 8 tests, but it silently fails so that we don't get the big red X. The one you noted happens about 1 in every 50 tests and seems to throw an alert that the password was not long enough (not silently failing this one which is why it give us the big X). When it happens again, will get a screenshot of it.

**Comment by bradymiller** [comment]
ok, was able to capture the elusive 1 in 50 test error out here.here it is:
```
Create Staff (OpenEMR\Tests\E2e\CreateStaff)
 ✘ Check add user
   │
   │ Facebook\WebDriver\Exception\UnexpectedAlertOpenException: unexpected alert open: {Alert text : Password too short. Minimum characters required: 9}
   │   (Session info: headless chrome=83.0.4103.116)
   │
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/Exception/WebDriverException.php:135
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/Remote/HttpCommandExecutor.php:371
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:612
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/Remote/RemoteWebDriver.php:233
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/WebDriverExpectedCondition.php:139
   │ /usr/share/nginx/html/openemr/vendor/php-webdriver/webdriver/lib/WebDriverWait.php:54
   │ /usr/share/nginx/html/openemr/vendor/symfony/panther/src/Client.php:331
   │ /usr/share/nginx/html/openemr/tests/Tests/E2e/Pages/UsersTab.php:60
   │ /usr/share/nginx/html/openemr/tests/Tests/E2e/CreateStaffTest.php:43
   │
```

**Comment by bradymiller** [comment]
Do note the password being used is fine:
https://github.com/openemr/openemr/blob/master/tests/Tests/E2e/Pages/UsersTab.php#L45

These intermittent e2e issues can be tough to debug, although i am guessing by figuring out how to fix them, it will improve the code.

**Pull request closed without merging**